src/RIO/Yesod.hs

```diff
@@ -0,0 +1,20 @@
+module RIO.Yesod
+  ( module RIO
+  ) where
+
+import RIO
```

Comment:
I wonder if it would be possible to provide a `HasLogFunc` instance on the `App` type instead.

Reply:
Let me get back to you n that — probably on Friday.